xdsl/interpreters/builtin.py

```diff
@@ -86,7 +86,7 @@ def dense_int_or_fp_elements_value(
         attr: Attribute,
         type_attr: builtin.MemRefType[Any],
     ) -> ShapedArray[Any]:
-        assert isinstance(attr, builtin.DenseIntOrFPElementsAttr)
+        assert isa(attr, builtin.DenseIntOrFPElementsAttr)
```

Comment:
is this required? does `isattr` work instead by any chance?

Reply:
`isattr` would just call `isa` right?

Reply:
Yeah on second thought this is fine.